policy/modules/system/unconfined.if

```diff
@@ -51,7 +51,7 @@ interface(`unconfined_domain_noaudit',`
 	allow $1 self:file rw_file_perms;
 
 	# io_uring
-	allow $1 self:anon_inode { create map read write };
+	allow $1 self:anon_inode { manage_file_perms mounton quotaon relabel_file_perms watch watch_mount watch_reads watch_sb watch_with_perm };
```

Comment:
Anonymous inodes are used by more than just io_uring, see the recent secretmem discussion, so it might be a good idea to update that comment while you are at it.